test/ia-account-settings.test.ts

```diff
@@ -0,0 +1,132 @@
+import { html, fixture, expect } from '@open-wc/testing';
+import { stub } from 'sinon';
+import type { IAAccountSettings } from '../src/ia-account-settings';
+import '../src/ia-account-settings';
+import MailingLists from '../src/assets/config';
```

Comment:
Suggestion: uncapitalized `mailingLists`, since it is a normal value and not a class.